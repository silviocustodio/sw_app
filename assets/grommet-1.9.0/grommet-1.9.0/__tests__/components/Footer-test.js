// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../../src/js/components/Footer';

// needed because this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

describe('Footer', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Footer>Testing</Footer>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('renders a class name', () => {
    const component = renderer.create(
      <Footer className="test">Testing</Footer>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('renders microdata properties', () => {
    const component = renderer.create(
      <Footer itemScope={true} itemType="http://schema.org/WPFooter"
        itemProp="test">Testing</Footer>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
