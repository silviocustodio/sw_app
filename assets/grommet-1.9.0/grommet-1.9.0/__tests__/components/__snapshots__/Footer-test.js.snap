// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer has correct default options 1`] = `
<footer
  className="grommetux-box grommetux-box--direction-row grommetux-box--align-center grommetux-box--pad-none grommetux-footer"
  id={undefined}
  onClick={undefined}
  role={undefined}
  style={Object {}}
  tabIndex={undefined}
>
  Testing
</footer>
`;

exports[`Footer renders a class name 1`] = `
<footer
  className="grommetux-box grommetux-box--direction-row grommetux-box--align-center grommetux-box--pad-none grommetux-footer test"
  id={undefined}
  onClick={undefined}
  role={undefined}
  style={Object {}}
  tabIndex={undefined}
>
  Testing
</footer>
`;

exports[`Footer renders microdata properties 1`] = `
<footer
  className="grommetux-box grommetux-box--direction-row grommetux-box--align-center grommetux-box--pad-none grommetux-footer"
  id={undefined}
  itemProp="test"
  itemScope={true}
  itemType="http://schema.org/WPFooter"
  onClick={undefined}
  role={undefined}
  style={Object {}}
  tabIndex={undefined}
>
  Testing
</footer>
`;
